internal/services/apimanagement/api_management_diagnostic_resource.go

```diff
@@ -259,7 +272,7 @@ func resourceApiManagementDiagnosticRead(d *pluginsdk.ResourceData, meta interfa
 				d.Set("backend_request", nil)
 				d.Set("backend_response", nil)
 			}
-			format := string(diagnostic.OperationNameFormatName)
+			format := ""
```

Comment:
Same here, pre 4.0 we should still be defaulting this in the read.

Reply:
Fixed.